Handleiding/Declaraties/Declaraties.rst

```diff
@@ -0,0 +1,82 @@
+
+7) **Factuurreferentie**: Voer indien nodig enige referentietekst in die moet worden opgenomen bij de uitgave.
+
+8) **Uitgavedatum**: Gebruik de kalendermodule om de datum in te voeren waarop de uitgave is gedaan. Gebruik de pijlen < (links) en > (rechts) om naar de juiste maand te navigeren, klik vervolgens op de specifieke dag om deze te selecteren.
+
+9) **Rekening**: Selecteer de kostenrekening waarop deze uitgave moet worden geboekt uit het dropdown-menu.
+
+10) **Klant om door te berekenen**: Als de uitgave door een klant moet worden betaald, selecteer dan de klant die voor deze uitgave moet worden gefactureerd uit het dropdown-menu. Alle verkooporders in de dropdown-lijst tonen zowel de klant als het bedrijf waarvoor de verkooporder is geschreven, maar nadat de uitgave is opgeslagen, verdwijnt de klantnaam en blijft alleen het bedrijf zichtbaar.
+
+11) **Analytisch**: Selecteer via het dropdown-menu de rekening(en) waarop de uitgave moet worden geboekt, voor Projecten, Afdelingen, of beide. Indien nodig kunnen meerdere rekeningen worden vermeld voor elke categorie. Pas het percentage aan voor elke analytische rekening door de waarde in te typen naast de rekening.
+
+12) **Notities...**: Voer eventuele notities in die nodig zijn om de uitgave te verduidelijken.
+
+.. image:: Declaraties_Media/Declaraties001.png
+
+
+**Voeg een bonnetje toe**
+--------------------------
+
+Nadat de uitgave is aangemaakt, is de volgende stap het toevoegen van een bonnetje. Klik op de knop Voeg bonnetje toe, en een verkenner verschijnt. Navigeer naar het bonnetje dat je wilt toevoegen en klik op Openen. Het nieuwe bonnetje wordt geregistreerd in de chatter, en het aantal bonnetjes verschijnt naast het 📎 (paperclip) pictogram onder het uitgavenformulier. Er kan meer dan één bonnetje aan een individuele uitgave worden toegevoegd, indien nodig. Het aantal bonnetjes dat aan de uitgave is toegevoegd, wordt weergegeven op het paperclip-pictogram.
+
+.. image:: Declaraties_Media/Declaraties002.png
+
+**Nieuwe uitgaven aanmaken vanuit een gescand bonnetje**  
+----------------------------------------------------------
+
+In plaats van alle informatie voor een uitgave handmatig in te voeren, kunnen uitgaven worden aangemaakt door een PDF-bonnetje te scannen.  
+Klik eerst in het hoofdoverzicht van de **Uitgaven**-app (dit overzicht kan ook worden geopend via **Uitgaven-app ‣ Mijn Uitgaven ‣ Mijn Uitgaven**), op **Scan**, en een verkenner verschijnt. Navigeer naar het bonnetje dat je wilt uploaden, klik erop om het te selecteren, en klik vervolgens op **Openen**.
```

Comment:
` op **Scan** ` change to `klik daarna op **Scan**`
`en een verkenner verschijnt` this is translated from explorer I guess. Windows Explorer is a windows-only application so this can be different on Ubuntu Desktop e.g. maybe you can change it to something more generic like "file manager" or avert from the word all together.